thirdeye/thirdeye-frontend/app/pods/components/filter-bar/component.js

```diff
@@ -97,7 +97,16 @@ export default Ember.Component.extend({
   entitiesObserver: Ember.observer(
     'entities',
     function() {
-      const filterBlocks = this.get('config');
+      const { config: filterBlocks, entities } = this.getProperties('config', 'entities');
+
+      // When entities are changed (i.e. from user interacting with the selection form), recompute urns to cache
+      filterBlocks.forEach(block => {
```

Comment:
can `filterBlocks` be null or undefined here?

Reply:
No, since it's a guarantee that the `config` file that gets passed is not null.